makePoplogCommander.sh

```diff
@@ -181,10 +342,20 @@ poplog (pop11|prolog|clisp|pml) [OPTION]... [VEDCOMMAND] [FILE]...
      %nobanner
          Suppress printing of the Poplog banner.
 
-    These interpreter commands have four different argument-patterns:
+    These interpreter commands (pop11, prolog, clisp, pml) have four different 
```

Comment:
A note to future us: we should write some systems tests to check these command patterns work for all different languages.

Reply:
Trello task added